internal/core/core.go

```diff
@@ -25,7 +28,29 @@ func BuildRootCmd(conf *config.Config) cobra.Command {
 				return err
 			}
 
-			terminal.ForceColours(conf.GlobalFlags.Colors)
+			terminal.ForceColours(conf.GlobalFlags.Colors && terminal.IsStdoutTerminal())
+
+			// Set up flume
+			flume.SetOut(os.Stderr)
+			// TODO: should we make the level configurable?
+			logLvl := flume.DebugLevel
+			if !conf.GlobalFlags.Debug {
+				// not debugging, no log output!
+				logLvl = flume.OffLevel
+			}
```

Comment:
Not really if debug log levels even make sense in this context? Could we just make do with one level of logging - it would make the executor side simpler (only Debug/DebugLog method)?

Reply:
agreed, it makes no sense to have different levels if `--debug` enables the logging in the first place.